activemodel/lib/active_model/validations/validates.rb

```diff
@@ -20,6 +20,8 @@ module ClassMethods
       #   validates :first_name, length: { maximum: 30 }
       #   validates :age, numericality: true
       #   validates :username, presence: true
+      #   validates :username, absence: true
+      #   validates :email, uniqueness: true
```

Comment:
We don't include uniqueness because it is not part of Active model. Can you remove it?

Reply:
Did I do notify you correctly? I changed the commit as requested but nothing has happened since. I realise you're busy.

The docs are still missing `absence` for `validates.rb`. Actually, the comment should have been at the top of the validates examples as they are all alphabetical. But I won't make any further change without comment.

Reply:
If you want to move the comment to the top, go ahead - otherwise this is ready to be merged. 👍

Reply:
I have lost the original fork and In trying to get it locally on my machine I've managed to close this fork - all in all a remarkable piece of work by me.

Reply:
I think it's technically possible to recover this pull request - from [this page](https://github.com/notapatch/rails/tree/d8866b7b5a97d57eed813186e6e00b3b64423966), you should be able to make a new branch on your fork that contains the old commit, fetch it locally, force push it back to the master branch of your fork, reopen the pull request, and _then_ force push the branch again with the change you wanted to make.

It might be simpler to just recreate the change locally and open a new pull request, though.

Reply:
Thanks @eugeneius, I went for the simpler way. 

https://github.com/rails/rails/pull/38017